Gruntfile.js

```diff
@@ -0,0 +1,19 @@
+module.exports = function(grunt) {
```

Comment:
Could you get grunt to work? If couldn't, then there's no reason to keep it in the project, same with nodeJS at all.